db/migrate/20200904113940_create_categorizations.rb

```diff
@@ -0,0 +1,10 @@
+class CreateCategorizations < ActiveRecord::Migration[5.2]
+  def change
+    create_table :categorizations do |t|
+      t.references :item, foreign_key: true
+      t.references :category, foreign_key: true
```

Comment:
in case of add reference always prefer give custom name to it. Good practice. Please write new migration and do this.

Reply:
I will in mind for my next tasks.